lib/notifiers/notifiers.go

```diff
@@ -49,7 +49,9 @@ const (
 
 var (
 	// Set of allowed notifier Config `apiVersions`.
-	allowedYAMLAPIVersions = map[string]bool{"cloud-build-notifiers/v1alpha1": true}
+	allowedYAMLAPIVersions = map[string]bool{
+		"github.com/GoogleCloudPlatform/cloud-build-notifiers/v1": true,
+	}
```

Comment:
This is going to break any user that updates on their next deploy. At minimum, consider adding instructions for how to update, and any differences between v1alpha1 and v1.

Alternatively, any reason not to continue to support v1alpha1 for the time being?

Reply:
While there's no difference in the v1 and v1alpha1 config format, Alpha users are aware that there can be breaking changes at any time (e.g. we only started enforcing apiVersion recently). If users truly care about this (they shouldn't, because Alpha), we have the < 1.x.y versions still available.

Reply:
👍  The only other thing I'd say is make sure in release notes that there's a clear breaking change warning. I often find it's useful to edit commits / PR messages with a similar warning, but up to you if you want to do that or just in the release notes.

Reply:
@navierula  has this covered in the forthcoming GA docs (we only mention `c-b-n/v1`), but I'll make a big note in this PR.